### PR TITLE
To account for oneMKL beta10 requirements

### PR DESCRIPTION
**Pull request by jedxops**
# Description

Changed rng.hpp header include location.

Fixes # (issue) 

N/A

## Type of change

Source file change to include statement.

Please delete options that are not relevant. Add a 'X' to the one that is applicable. 

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [X] Command Line
- [X] Visual Studio

# Checklist for Moving samples:
Links and Details can be found in the samples WG Teams Files. 

- [X] Review sample design with domain reviewers https://github.com/oneapi-src/oneAPI-samples/wiki/Reviewers-and-Domain-Experts 
- [X] Implement coding guidelines and ensure code quality.
- [X] Adhere to sample.json specification. https://github.com/oneapi-src/oneAPI-samples/wiki/sample-json-specification
- [X] Run jsonlint on sample.json to verify json syntax. www.jsonlint.com
- [X] Adhere to readme template 
- [X] Ensure/create CI test configurations for sample (ciTests field) https://github.com/oneapi-src/oneAPI-samples/wiki/sample-json-ci-test-object
- [X] Enforce format via clang-format config file
- [X] Review DPC++ code with Paul Peterseon. (GitHub User: pmpeter1)
- [X] Review readme with Tom Lenth or Joe Oster. (GitHub User: JoeOster)
- [X] Tested using Dev Cloud when applicable
- [X] Implement fixes for ONSAM Jiras
- [X] If you have new dependencies/binaries, inform Samples Project Manager Swapna R Dontharaju (@srdontha) or @JoeOster

### CODE REVIEW

File: DirectProgramming/DPC++/StructuredGrids/particle-diffusion/src/motionsim_kernel.cpp [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+// =============================================================
+
+// motionsim_kernel:
+// Intel® oneAPI DPC++ Language Basics Using a Monte Carlo
+// Simulation (Device)
+//
+
+#include "motionsim.hpp"
+namespace oneapi {}
+using namespace oneapi;
```
Comment (JoeOster) [inline DirectProgramming/DPC++/StructuredGrids/particle-diffusion/src/motionsim_kernel.cpp:14]: @pmpeter1 - Is there a specific rule against "using oneapi" in the cpp code?

**Review by JoeOster — APPROVED** [review]
approving to launch ci

File: DirectProgramming/DPC++/StructuredGrids/particle-diffusion/src/motionsim_kernel.cpp (hunk 1 above)
Comment (pmpeter1) [reply DirectProgramming/DPC++/StructuredGrids/particle-diffusion/src/motionsim_kernel.cpp:14]: The sample style guidelines say it's OK (and often preferred) to use "using" in the .cpp files explicitly.   It should not be in a header file.     This is to make it obvious you are introducing this short-cut, and to keep the code simpler.

I'm not exactly sure why we are introducing an empty namespace and then opening it here.   So the question is why like 13?

Comment (petercad) [reply DirectProgramming/DPC++/StructuredGrids/particle-diffusion/src/motionsim_kernel.cpp:14]: This looks like a temporary measure to support beta08 oneMKL (where APIs are in the mkl namespace) and beta09 oneMKL (where APIs are in oneapi::mkl) simultaneously.

Comment (jedxops) [reply DirectProgramming/DPC++/StructuredGrids/particle-diffusion/src/motionsim_kernel.cpp:14]: Honestly, I have no idea as to the implications of the syntax configuration of lines 13 and 14, I was just following what Alberto Villarreal did in his version to declare the namespace

Comment (jedxops) [reply DirectProgramming/DPC++/StructuredGrids/particle-diffusion/src/motionsim_kernel.cpp:14]: I will be happy to change it if there is a proposed, preferred way of declaration.

**Pull request merged**